altdoc/about.md

```diff
@@ -0,0 +1,38 @@
+## Developers
+
+The primary developer of the upstream Polars project is Ritchie Vink
+([@ritchie46](https://github.com/ritchie46)). This R package is
+maintained by [@eitsupi](https://github.com/eitsupi) and [Etienne Bacher](https://github.com/etiennebacher). The original R package was created by Søren Welling
+([@sorhawell](https://github.com/sorhawell)).
+
+Development occurs on Github: https://github.com/pola-rs/r-polars
+
+Consider joining our [Discord](https://discord.com/invite/4UfP5cfBE7)
+(subchannel) for additional help and discussion.
+
+**'Polars'** is a query engine/syntax for multiple languages. The R specific implementation
+can be referred to as **r-polars** and is derived from **rust-polars**.
```

Comment:
The terms need to be changed to Rust API and R API.

Reply:
Done